sympy/matrices/tests/test_matrices.py

```diff
@@ -2142,6 +2142,11 @@ def test_is_Identity():
     assert not ones(3).is_Identity
     # issue 6242
     assert not Matrix([[1, 0, 0]]).is_Identity
+    # Pull Request 8854 (Some tests for the "is_Identity" property of Sparse Matrix)
```

Comment:
you can go ahead and write "issue 8854" and it will reference to the PR if someone tries to look up that issue. You can delete the parenthetical comment as that's apparent from the lines added (and the PR itself).

After making those changes, could you squash these commits into a single one and then just do "git push -f laituan245 master" and it will update that PR -- no need to open a new PR.